### ISSUE TITLE
Fullscreen mode: notice shows the wrong message

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#69303]
### Description

In the Post editor, when switching fullscreen mode off and on _via the keyboard shortcut_, a snackbar notice appears and a speak message is sent to the ARIA live region but the message is the wrong one. It should be swapped.

Video to illustrate:

https://github.com/user-attachments/assets/eba0967c-5c4f-46b2-938a-90cdcf29bf5a

To my understanding, the notice was introduced in https://github.com/WordPress/gutenberg/pull/66405

### Step-by-step reproduction instructions

- Edit a post
- Use the fullscreen mode keyboard shortcut. On macOS it's `Shift+Option+Command+F`.
- Observe the snackbar notice says 'activated' when deactivating and 'deactivated' when activating.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure
